src/main/java/org/cryptacular/CiphertextHeaderV2.java

```diff
@@ -253,8 +253,10 @@ public static CiphertextHeaderV2 decode(final InputStream input, final Function<
    *
    * @param  input  Input stream.
    * @param  output  Output buffer.
+   *
+   * @throws  StreamException  on stream IO errors.
    */
-  private static void readInto(final InputStream input, final byte[] output)
+  private static void readInto(final InputStream input, final byte[] output) throws StreamException
```

Comment:
I only meant for you to add the javadoc. I don't think it's necessary to declare an unchecked exception here. In fact, it's probably a detriment given our versioning policy.